.github/workflows/pr.yaml

```diff
@@ -110,7 +110,7 @@ jobs:
         run: |
           # We need a Drupal project to run drupal-check (cf. https://github.com/mglaman/drupal-check#usage)
           # Install Drupal
-          composer --no-interaction create-project drupal/recommended-project:^9 --stability=dev drupal
+          composer --no-interaction create-project drupal/recommended-project:^10 --stability=dev drupal
```

Comment:
I've found a cleaner way to do this and we should consider updating this all over the place; see https://github.com/os2forms/os2forms#code-analysis (and https://github.com/OS2Forms/os2forms/blob/develop/scripts/code-analysis) for details.

Reply:
Changed to use script.